### DISCUSSION TITLE
Companion for EdgeTX Ver 2.8

### DISCUSSION THREAD

**Discussion by Carl-RC** [community/community#40634] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When I added a new Radio (Jumper T18) to Companion Version for EdgeTX 2.8, the label assigned to the TX was "**New Radio**" and not "**Jumper T18**"!  In the past, the label matched the radio.  What happened or am I missing something somewhere?
[REDACTED DUE TO PII]

**Comment by ossanna16** [comment]
Hi there @Carl-RC and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
